pontoon/settings/base.py

```diff
@@ -753,22 +753,69 @@ def _default_from_email():
 )
 
 # Logging
+# Get environment variables
+LOG_TO_FILE = os.getenv("LOG_TO_FILE", "False") == "True"
+
+# Ensure the logs directory exists
+if LOG_TO_FILE:
+    log_dir = path("logs")
+    os.makedirs(log_dir, exist_ok=True)
+
+# Define handlers
+console_handler = {
+    "class": "logging.StreamHandler",
+    "formatter": "verbose",
+}
+
+django_file_handler = {
+    "class": "logging.handlers.RotatingFileHandler",
+    "filename": path("logs", "django_debug.log"),
+    "maxBytes": 1024 * 1024 * 2,  # 2 MB
+    "backupCount": 3,
+    "formatter": "verbose",
+}
+
+pontoon_file_handler = {
+    "class": "logging.handlers.RotatingFileHandler",
+    "filename": path("logs", "pontoon_debug.log"),
+    "maxBytes": 1024 * 1024 * 2,  # 2 MB
+    "backupCount": 3,
+    "formatter": "verbose",
+}
+
+# Define logging configuration
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
-    "handlers": {"console": {"class": "logging.StreamHandler"}},
+    "handlers": {
+        "console": console_handler,
+    },
```

Comment:
Please restore this change. It doesn't seem needed to fix #3260. Then we can also remove `console_handler` above.